### PR TITLE
x11-terms/mlterm: allow wayland without X

### PR DESCRIPTION
**Pull request by vaukai**
Package-Manager: Portage-3.0.13, Repoman-3.0.2
Signed-off-by: Volkmar W. Pogatzki <gentoo@pogatzki.net>

### CODE REVIEW

File: x11-terms/mlterm/mlterm-3.9.1.ebuild [hunk 1]
```diff
@@ -12,12 +12,11 @@ SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
 LICENSE="BSD"
 SLOT="0"
 KEYWORDS="~amd64 ~ppc ~ppc64 ~x86"
-IUSE="bidi brltty cairo canna debug fbcon fcitx freewnn gtk gtk2 harfbuzz ibus libssh2 m17n-lib nls regis scim skk static-libs uim utempter wayland xft"
+IUSE="X bidi brltty cairo canna debug fbcon fcitx freewnn gtk gtk2 harfbuzz ibus libssh2 m17n-lib nls regis scim skk static-libs uim utempter wayland xft"
 REQUIRED_USE="gtk2? ( gtk )"
```
Comment (ionenwks) [inline x11-terms/mlterm/mlterm-3.9.1.ebuild:16]: Need `|| ( X fbcon wayland )` and a default (`+X` would make sense to preserve old behavior), if all are unset it tries to use automagic.

**Review by ionenwks — COMMENTED** [review]
Being conditional will have some implications, for now I see:
- USE=gtk's mlimgloader has automagic on libX11, can build without but will use if found.
- USE=scim fails to build if Xlib.h doesn't exist

**Comment by ionenwks** [comment]
>     * USE=gtk's mlimgloader has automagic on libX11, can build without but will use if found.
Solution to this one seems to be to pass --without-x when USE=-X

File: x11-terms/mlterm/mlterm-3.9.1.ebuild (hunk 1 above)
Comment (vaukai) [reply x11-terms/mlterm/mlterm-3.9.1.ebuild]: Done

**Comment by vaukai** [comment]
> > ```
> > * USE=gtk's mlimgloader has automagic on libX11, can build without but will use if found.
> > ```
> 
> Solution to this one seems to be to pass --without-x when USE=-X

Oops, how could I have missed this? 
Done.

**Comment by vaukai** [comment]
Ping

**Pull request closed without merging**